.github/workflows/ci.yml

```diff
@@ -34,6 +34,7 @@ jobs:
         os:
           - 'debian-10'
           - 'centos-8'
+          - 'centos-6'
```

Comment:
CentOS 6 EOL is 11/30/2020 -- I'm not sure adding this to the test matrix would be beneficial at this juncture.

Reply:
Also as above.

Reply:
Removed.